mods/default/init.lua

```diff
@@ -20,6 +20,10 @@ minetest.register_on_joinplayer(function(player)
 			listcolors[#00000069;#5A5A5A;#141318;#30434C;#FFF] ]]
 	local name = player:get_player_name()
 	local info = minetest.get_player_information(name)
+	if not info then
+		minetest.log("warning", "player " .. name .. " disappeared")
+		return
+	end
 	if info.formspec_version > 1 then
```

Comment:
Just 
`if info and info.formspec_version > 1 then`

Reply:
This would hide that something fishy is going on. I have to figure out whether returning from the handler is the best thing to do, though.

Reply:
MoNTE488, my PR #2586 did exactly that and was disapproved.